scripts/windows-native-ci.ps1

```
@@ -5388,14 +5388,47 @@ function Stop-StateProcesses([string]$Root) {
 function Get-WindowsNativeCaptureFiles([string]$Root) {
     if (-not (Test-Path -LiteralPath $Root)) { return @() }
+    try {
+        $rootItem = [IO.DirectoryInfo](Get-Item -LiteralPath $Root -Force -ErrorAction Stop)
+    } catch {
+        return @()
+    }
+    if (Test-WindowsNativeReparsePoint $rootItem) { return @() }
+
+    $pending = [Collections.Generic.Queue[IO.DirectoryInfo]]::new()
+    $pending.Enqueue($rootItem)
+    $matches = [Collections.Generic.List[IO.FileInfo]]::new()
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)